.github/workflows/formatting.yml

```diff
@@ -0,0 +1,118 @@
+name: Code Formatting Check
+
+on:
+  workflow_run:
+    workflows: [ "Engine CI (legacy)", "Engine CI (new)" ]
+    branches: [ main ]
+    types:
+      - completed
+  push:
+    branches: [develop, "release/*"]
+  pull_request:
+    branches: ["*"]
```

Comment:
Please check if removing the branches key altogether works. Likely `*` does not match branch names with `/`.